apps/vue-plugin-e2e/tests/vue-plugin.test.ts

```diff
@@ -21,6 +21,10 @@ describe('vue-plugin e2e', () => {
       Snapshots:   0 total
     `);
 
+    const e2eResult = await runNxCommandAsync(`e2e ${appName}-e2e --headless`);
+    expect(e2eResult.stdout).toContain('✓ should display welcome message');
+    expect(e2eResult.stdout).toContain('✔  All specs passed!');
```

Comment:
Remove the check mark. I think we should avoid asserting against those sort of special characters.